pytorch_lightning/plugins/precision/deepspeed_precision.py

```diff
@@ -33,28 +37,30 @@ def pre_optimizer_step(
 
     def backward(
         self,
-        lightning_module: LightningModule,
+        model: 'LightningModule',
```

Comment:
isn't this API change?

Reply:
It was named model in all the other plugins and I checked, It is never called by name. 
But if we don't do this, MypY complains, that the signature does not match with base class since it has different parameters